python/docs/source/index.md

```diff
@@ -3,94 +3,119 @@ hide-toc: true
 html_theme.sidebar_secondary.remove: true
 title: Kaskada Timestreams
 ---
+# Introduction
 
 <div class="px-4 py-5 my-5 text-center">
     <img class="d-block mx-auto mb-4" src="_static/kaskada.svg" alt="" width="auto">
-    <h1 class="display-5 fw-bold">Event-processing for AI applications.</h1>
+    <h1 class="display-5 fw-bold">Real-Time AI without the fuss.</h1>
     <div class="col-lg-7 mx-auto">
-      <p class="lead mb-4">Next-generation, real-time and historic event processing.
+      <p class="lead mb-4">Kaskada is a simple streaming engine that connects AI models to real-time & historical data.
```

Comment:
Any risk that "simple" is received as "limited"?

Overall, I think emphasizing the use for Real-Time AI is good, but I worry that this loses the explanation of what it is (event-processing / temporal queries / compute / etc.)

Reply:
Changed "simple" to "next-generation".